chart/templates/mayastor/agents/core/agent-core-service.yaml

```diff
@@ -11,3 +11,5 @@ spec:
   ports:
     - name: grpc
       port: 50051
+    - name: ha-cluster
+      port: 11500
```

Comment:
Should we use 50052?

Reply:
That would need changes from control-plane side as well. But yeah we can do it if needed

Reply:
Why would it need control-plane changes?

Reply:
Ohh it won't, I was thinking we need to change the constants in the control-plane but that is independent of this. Should I make the change then?

Reply:
Up to you but I think so, would be good to keep the ports together, will also be easier to remember

Reply:
Done. Changed the node-agent's server port to be 50053 as well. Also added jaeger arg to node-agent

Reply:
Nice and good catch on the node-agent jaeger!